pycyphal/transport/can/media/socketcand/_socketcand.py

```diff
@@ -0,0 +1,280 @@
+        self._port = port
+        self._can_channel = channel
+
+        self._closed = False
+        self._maybe_thread: typing.Optional[threading.Thread] = None
+        self._rx_handler: typing.Optional[Media.ReceivedFramesHandler] = None
+        # This is for communication with a thread that handles the call to _bus.send
+        self._tx_queue: queue.Queue[_TxItem | None] = queue.Queue()
+        self._tx_thread = threading.Thread(target=self._transmit_thread_worker, daemon=True)
+
+        try:
+            bus = can.ThreadSafeBus(
+                interface=self._iface,
+                host=self._host,
+                port=self._port,
+                channel=self._can_channel,
+            )
+
+            self._bus: can.ThreadSafeBus = bus
+        except can.CanError as ex:
+            raise InvalidMediaConfigurationError(f"Could not initialize PythonCAN: {ex}") from ex
+        super().__init__()
+
+    @property
+    def interface_name(self) -> str:
+        return self._iface
+
+    @property
+    def channel_name(self) -> str:
+        return self._can_channel
+
+    @property
+    def host_name(self) -> str:
+        return self._host
+
+    @property
+    def port_name(self) -> int:
+        return self._port
+
+    # Python-CAN's wrapper for socketcand does not support FD frames, so mtu will always be 8 for now
+    @property
+    def mtu(self) -> int:
+        return 8
+
+    @property
+    def number_of_acceptance_filters(self) -> int:
+        """
+        The value is currently fixed at 1 for all interfaces.
+        TODO: obtain the number of acceptance filters from Python-CAN.
+        """
+        return 1
+
+    def start(self, handler: Media.ReceivedFramesHandler, no_automatic_retransmission: bool) -> None:
+        self._tx_thread.start()
+        if self._maybe_thread is None:
+            self._rx_handler = handler
+            self._maybe_thread = threading.Thread(
+                target=self._thread_function, args=(asyncio.get_event_loop(),), name=str(self), daemon=True
+            )
+            self._maybe_thread.start()
+            if no_automatic_retransmission:
+                _logger.info("%s non-automatic retransmission is not supported", self)
+        else:
+            raise RuntimeError("The RX frame handler is already set up")
+
+    def configure_acceptance_filters(self, configuration: typing.Sequence[FilterConfiguration]) -> None:
+        if self._closed:
+            raise ResourceClosedError(repr(self))
+        filters = []
+        for f in configuration:
+            d = {"can_id": f.identifier, "can_mask": f.mask}
+            if f.format is not None:  # Per Python-CAN docs, if "extended" is not set, both base/ext will be accepted.
+                d["extended"] = f.format == FrameFormat.EXTENDED
+            filters.append(d)
+        _logger.debug("%s: Acceptance filters activated: %s", self, ", ".join(map(str, configuration)))
+        self._bus.set_filters(filters)
```

Comment:
```suggestion
        self._bus.set_filters(filters)
        _logger.debug("%s: Acceptance filters activated: %s", self, ", ".join(map(str, configuration)))
```

Either this, or replace the log string to state that the filters are being activated.